### PR TITLE
Release 1.3.0 with uproot 4.0 and awkward 1.0

### PR DESCRIPTION
**Pull request by eduardo-rodrigues**

### CODE REVIEW

**Comment by eduardo-rodrigues** [comment]
@jpivarski, can you in particular check the updates related to the uproot and awkward series? In a tmp docker container all seems fine, still. Thanks.

**Review by jpivarski — APPROVED** [review]
These are the correct versions for `awkward0`, `awkward`, `uproot3`, `uproot`, and `uproot3-methods`. It also correctly removes `uproot-methods` (no number), which should not be used anymore.

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  rev: "0.45"
+  hooks:
+  - id: check-manifest
+    stages: [manual]
+- repo: https://github.com/pre-commit/mirrors-mypy
+  rev: v0.790
+  hooks:
+  - id: mypy
+    files: src
+    additional_dependencies: [attrs==19.3.0]
```
Comment (henryiii) [inline .pre-commit-config.yaml:32]: I don't think we need this? scikit-hep doesn't use attires? This should be a list of the minimal requirement to do static type checking (usually just the packages that support type checking)

Actually, more fundamentally, this is only looking at files in `/src`, which there are none, so I'd probably remove this check until we support some type checking (and then the files setting would need to be correct). Only useful if there's some code to check.

Comment (eduardo-rodrigues) [reply .pre-commit-config.yaml:32]: Fair enough, I will remove these lines.

Comment (eduardo-rodrigues) [reply .pre-commit-config.yaml]: I had simply copied and pasted the file from Particle, and not everything was relevant. Fair enough. Thanks.

**Comment by henryiii** [comment]
I've added an explicit skip list for items not in the MANIFEST - please make sure that none of those things are supposed to go in.

**Comment by eduardo-rodrigues** [comment]
Thanks a lot. All ready?

**Review by henryiii — APPROVED** [review]

**Pull request merged**